### PR TITLE
Update return type of NI-FGEN datetime-related functions

### LINKED ISSUE #1716: The return value of `get_date_and_time()` related function(s) across drivers should be consistently named
### Description of issue

Currently, the return value of `get_date_and_time()` related function(s) is correctly named "last_cal_datetime" in NI-SCOPE, but incorrectly named "month" in NI-DCPower, NI-DMM, NI-FAKE and NI-FGEN. They should be made consistent.

This issue is mentioned in #1707.

### PR DESCRIPTION
**Pull request by ayesham-729**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

~~- [ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?

This pull request fixes a documentation issue where some functions in NI-FGEN that returned calibration dates returned `month` instead of `last_cal_datetime`. This has been updated to be consistent with SCOPE and DC-Power. 

### List issues fixed by this Pull Request below, if any.

Part of fix for issue "The return value of get_date_and_time() related function(s) across drivers should be consistently named"

### What testing has been done?

Confirmed that generated files show `last_cal_datetime` as the return value instead of `month`.

### CODE REVIEW

**Comment by sbethur** [comment]
@ayesham-729, please update the PR title

**Comment by ayesham-729** [comment]
@sbethur do I need to update the changelog?

File: src/nifgen/metadata/functions.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FGEN API metadata version 21.5.0d106
+# This file is generated from NI-FGEN API metadata version 22.0.0d9999
```
Comment (marcoskirsch) [inline src/nifgen/metadata/functions.py:2]: Let's try to grab this from a real internal NI build artifact.
If you cannot, then let's temporarily override the name through the add-on.

File: generated/nifgen/nifgen/_library.py [hunk 2]
```diff
@@ -387,21 +387,21 @@ def niFgen_GetHardwareState(self, vi, state):  # noqa: N802
                 self.niFgen_GetHardwareState_cfunc.restype = ViStatus  # noqa: F405
         return self.niFgen_GetHardwareState_cfunc(vi, state)
 
-    def niFgen_GetLastExtCalLastDateAndTime(self, vi, month):  # noqa: N802
+    def niFgen_GetLastExtCalLastDateAndTime(self, vi, last_cal_datetime):  # noqa: N802
```
Comment (marcoskirsch) [inline generated/nifgen/nifgen/_library.py:390]: There is something wrong in our code generator that it creates this wrapper function for something that doesn't have a C function. @sbethur do we have any easy way to tell that the functions shouldn't be getting generated? `'python-only'`?

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: generated/nifgen/nifgen/_library.py (hunk 2 above)
Comment (sbethur) [reply generated/nifgen/nifgen/_library.py:390]: Yes, we should use 'python-only`, I'll fix it.

Comment (sbethur) [reply generated/nifgen/nifgen/_library.py:390]: Fixed it in internal metadata. @ayesham-729, please update this PR with the latest export of internal metadata.

**Comment by sbethur** [comment]
@ayesham-729, GitHub auto-closes an issue if it can find a string like "Fix #xyz" in the PR description. This is pretty cool, but in this case, we don't want that to happen since the referenced issue also talks about needing fixes in other nimi-python modules.

File: src/nifgen/metadata/functions.py (hunk 1 above)
Comment (ayesham-729) [reply src/nifgen/metadata/functions.py]: Updated

File: generated/nifgen/nifgen/_library.py (hunk 2 above)
Comment (ayesham-729) [reply generated/nifgen/nifgen/_library.py]: Updated

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**